playbooks/README.md

```diff
@@ -44,20 +44,22 @@ TODO: Describe
     cat ~/inventories/hosts
     [afs_kdcs]
     afs01.example.com
-    
+
     [afs_databases]
     afs01.example.com
     afs02.example.com
     afs03.example.com
-    
+
     [afs_fileservers]
     afs04.example.com
     afs05.example.com
     afs06.example.com
-    
+
     [afs_clients]
     afs[01:06].example.com
 
+    [afs_all]
```

Comment:
I feel this might be better named "afs_cell" since, that's what it is, all the hosts for a given cell.

Reply:
Not  a problem.. I'll change it and push a new PR